Fabric Editor/FabricApp.swift

```diff
@@ -67,3 +68,29 @@ struct AboutCommands: Commands {
         }
     }
 }
+
+struct ViewCommands: Commands {
+    @FocusedValue(\.centerOnSelectedNode) private var centerOnSelectedNode: (() -> Void)?
+    
+    var body: some Commands {
+        CommandGroup(after: .sidebar) {
+            Button("Center on Selected Node") {
+                centerOnSelectedNode?()
+            }
+            .keyboardShortcut("f", modifiers: .command)
+            .disabled(centerOnSelectedNode == nil)
+        }
+    }
+}
+
+// FocusedValue key for center on selected node action
+struct CenterOnSelectedNodeKey: FocusedValueKey {
+    typealias Value = () -> Void
+}
+
+extension FocusedValues {
```

Comment:
Is this strictly necessary? I think the focus value is for keyboard/device input, which in the canvas shouldnt be a thing unless youre interacting with the node via input device somehow? 

Do we need to track focus as well as selection? If so, cant we handle the binding using the .focus on the canvas view ? 

I ask because a node can be selected, but not focused (focus might be in the inspector side panel, or the node registry for example?)